test/Common/TestHelpers.cs

```diff
@@ -75,7 +79,7 @@ internal static class TestHelpers
                 options = new DurableTaskOptions();
             }
 
-            options.HubName = GetTaskHubNameFromTestName(testName, enableExtendedSessions);
+            options.HubName = exactTaskHubName ?? GetTaskHubNameFromTestName(testName, enableExtendedSessions);
```

Comment:
Unless I'm mistaken, this line is responsible for changing _all_ TaskHubNames, unless an exact TaskHub is specified. Since it's so consequential, I would recommend having short comment above it specifying what this line is doing, and why it's so important.